ValueSet/NHSDigital-message-events.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <ValueSet xmlns="http://hl7.org/fhir">
     <id value="England-message-events" />
-    <url value="https://fhir.nhs.uk/ValueSet/England-message-events" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-MessageEvents" />
     <version value="1.0.12" />
     <name value="NHSMessageEvents" />
```

Comment:
Doesn't this need to change to EnglandMessageEvents?

Reply:
The format in both UKCore & NHSE Dev. Guide is: https://fhir.hl7.org.uk/ValueSet/UKCore-[BusinessNames]
However, we have two NHS message events, and they are program inclined, BARS and EPS. So I am proposing to change their Meta's to indicate the originating programmes. Therefore, the canonical URL`s for ValueSets and CodeSystems will change as follows: 
1.https://fhir.nhs.uk/ValueSet/England-MessageEventsBARS
2.https://fhir.nhs.uk/ValueSet/England-MessageEventsEPS
I will let you know once it's done so that you can update [UKCoreMessageEvent](https://simplifier.net/hl7fhirukcorer4/valueset-ukcore-messageevent)